### PR TITLE
provide the possibility to use a secondary database for laravel tables

### PR DESCRIPTION
**Pull request by mszulik**
Docker features will be added in a follow-up branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,3 @@
...
-In order to ensure that the Laravel community is welcoming to all, please review and abide by the [Code of Conduct](https://laravel.com/docs/contributions#code-of-conduct).
-
-## Security Vulnerabilities
-
-If you discover a security vulnerability within Laravel, please send an e-mail to Taylor Otwell via [taylor@laravel.com](mailto:taylor@laravel.com). All security vulnerabilities will be promptly addressed.
-
-## License
-
-The Laravel framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
+Used to backup a non-Laravel project with the Laravel Protector package.
```
Comment (gael-connan-cybex) [inline README.md]: add a setup section

File: app/Console/Commands/CreateUser.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+class CreateUser extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'create:user
+                {protectorPublicKey : The Protector public key for the user.}
```
Comment (gael-connan-cybex) [inline app/Console/Commands/CreateUser.php:17]: -p

Comment (mszulik) [reply app/Console/Commands/CreateUser.php:17]: it's an argument.

File: README.md (hunk 1 above)
Comment (mszulik) [reply README.md]: done.

**Review by gael-connan-cybex — APPROVED** [review]
tested

**Pull request merged**